rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/DataUtility.java

```diff
@@ -216,10 +220,10 @@ public static List<JobInfo> toJobInfos(List<UserJobData> dataList) {
     }
 
     public static List<WorkflowExecutionTime>
-            toFilteredTopWorkflowsWithExecutionTime(List<WorkflowExecutionTimeData> filteredTopWorkflowsData) {
+            toWorkflowsExecutionTime(List<WorkflowExecutionTimeData> filteredTopWorkflowsData) {
         List<WorkflowExecutionTime> filteredTopWorkflows = new ArrayList<>(filteredTopWorkflowsData.size());
         for (WorkflowExecutionTimeData filteredTopWorkflow : filteredTopWorkflowsData) {
-            filteredTopWorkflows.add(toFilteredTopWorkflowWithExecutionTime(filteredTopWorkflow));
+            filteredTopWorkflows.add(toWorkflowExecutionTime(filteredTopWorkflow));
         }
         return filteredTopWorkflows;
```

Comment:
(To be confirmed with Fabien). ONLY IF  this function can operate on several data/jobs and IF it is frequently called, why not boosting it by using the Java 8 style:    filteredTopWorkflows = filteredTopWorkflowsData.stream().map(filteredTopWorkflow -> toWorkflowExecutionTime(filteredTopWorkflow); (if it is possible)?